fleet_notifications/notifications_client.py

```
@@ -42,6 +46,14 @@ def call_phone(self, phone_number: str, under_test: bool) -> None:
                 logger.error(f"An error occured while handling a call to number {phone_number} : {e}")
 
 
+    def _check_url_exists(self) -> bool:
+        try:
+            response = requests.get(self._url, timeout=5)
+            return response.status_code == 200
+        except requests.RequestException as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)